.github/workflows/create-release-on-tag.yml

```
@@ -22,15 +22,19 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: PXF Version ${{ github.ref }}
           body: |
-            ## 5.14.0 (06/30/2020)
+            ## 5.15.0 (08/19/2020)
 
             ### Enhancements:
 
-            - [#389](https://github.com/greenplum-db/pxf/pull/389) Avro: add compression
-            - [#391](https://github.com/greenplum-db/pxf/pull/391) Certify Oracle Enterprise Linux 7
+            - [#392](https://github.com/greenplum-db/pxf/pull/392) Add support for Avro BZip2 and XZ Compression Codecs
+            - [#395](https://github.com/greenplum-db/pxf/pull/395) Bump com.fasterxml.jackson.core:jackson-* version from 2.9.x to 2.11.0
+            - [#410](https://github.com/greenplum-db/pxf/pull/410) Allow skipping the header for *:text profiles
+            - [#423](https://github.com/greenplum-db/pxf/pull/423) Deprecate THREAD_SAFE custom option
```

can we use 421 instead?
```suggestion
            - [#421](https://github.com/greenplum-db/pxf/pull/421) Deprecate THREAD_SAFE custom option
```